src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/KeyValuePairExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen
+{
+    public static class KeyValuePairExtensions
+    {
+        // Explicit deconstruct required for older .NET frameworks
+        public static void Deconstruct<TKey, TValue>(this KeyValuePair<TKey, TValue> kvp, out TKey key, out TValue value)
+        {
+            key = kvp.Key;
+            value = kvp.Value;
+        }
+    }
```

Comment:
It would be better to only add this polyfill for the TFMs that need it. Also, can it be `internal`?